.github/workflows/xmake.yml

```
@@ -0,0 +1,45 @@
+        uses: actions/checkout@v4
+
+      - name: Setup xmake
+        uses: xmake-io/github-action-setup-xmake@v1
+        with:
+          xmake-version: branch@master
+          actions-cache-folder: ".xmake-cache"
+          actions-cache-key: ${{ matrix.os }}
+          package-cache: true
+          package-cache-key: ${{ matrix.os }}
+          build-cache: true
+          build-cache-key: ${{ matrix.os }}-${{ matrix.build_type }}
```

actionlint (1.7.8)

property "build_type" is not defined in object type {os: string}

(expression)